src/js/popup.js

```diff
@@ -277,7 +277,7 @@ const Logic = {
   },
 
   async showPanel(panel, currentIdentity = null, backwards = false, addToPreviousPanelPath = true) {
-    if ((!backwards && addToPreviousPanelPath) || !this._currentPanel) {
+    if ((!backwards && addToPreviousPanelPath) && this._currentPanel && this._currentPanel !== panel) {
```

Comment:
While the change seems to work in this specific step-to-reproduce, it introduces an issue in the conditional logic in the following scenario:

1. Open popup to show containerList
1. Click on the right arrow of the Personal container to go to the containerInfo panel
1. In containerInfo panel, click on "manage this container" button
1. In containerEdit panel, click on the back button to go to the previous panel containerInfo
1. In containerInfo panel, click on the back button to go to the previous panel containerList

On the last step, `backwards` value is set to `false` but it should be set to `true` since we clicked on the back button.